src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,93 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { getMovie } from '../../api';
 import './FindMovie.scss';
 
-export const FindMovie: React.FC = () => {
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+import { Movie } from '../../types/Movie';
+
+import { MovieCard } from '../MovieCard';
+
+interface Props {
+  onAdd: (movie: Movie) => void,
+}
+
+export const FindMovie: React.FC<Props> = ({ onAdd }) => {
+  const [query, setQuery] = useState('');
+  const [loadind, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
It's better to follow these rules with naming boolean variables to make your code cleaner and more understandable for other developers:
![image](https://user-images.githubusercontent.com/96409563/211368221-59f3f26a-e2e6-42d0-969a-6cb6b6bf2a12.png)